app.js

```diff
@@ -3,14 +3,19 @@ const app = express();
 const { getTopics } = require("./controllers/topics.controller");
 const { getApi } = require("./controllers/endpoints.controller");
 const { getArticleById } = require("./controllers/articles.controller");
-const { psqlErrorHandler, customErrorHandler, serverErrorHandler } = require("./error-handling");
+const {
+  psqlErrorHandler,
+  customErrorHandler,
+  serverErrorHandler,
+} = require("./error-handling");
+const { getArticles } = require("./controllers/articles.controller");
 
 app.use(express.json());
```

Comment:
avoid pasting in boiler plate code - you don't need express.json() yet. please remove

Reply:
I need it on the next task, can't remove